jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/Utf8Appendable.java

```diff
@@ -81,6 +81,7 @@ public abstract class Utf8Appendable implements CharsetStringBuilder
     };
 
     private int _codep;
+    private boolean _throwOnInvalid = true;
```

Comment:
Do we really need this?
If we keep it, it cannot be mutable, should be set as `final` in the constructor and never modified.

Reply:
I think not. See my other PR that is taking another approach to solving this problem.